src/custom/CortiAuth.ts

```diff
@@ -116,21 +200,45 @@ export class Auth extends FernAuth {
         }, requestOptions));
     }
 
+    /**
+     * Patch: PKCE-specific method for Authorization code flow
+     */
+    public getPkceFlowToken(
+        request: AuthorizationPkceServer,
+        requestOptions?: FernAuth.RequestOptions,
+    ): core.HttpResponsePromise<Corti.GetTokenResponse> {
+        const codeVerifier = request.codeVerifier || this.getCodeVerifier();
+        
+        if (!codeVerifier) {
+            throw new errors.CortiError({
+                message: "codeVerifier is required. Call authorizePkceUrl() first or provide it as parameter.",
+            });
```

Comment:
According to our documentation, `CortiError` is an error coming from API

> **CortiError**: Thrown when the API returns a non-success status code (4xx or 5xx response)

I think we should use `ParseError` for validation instead (that's what I do for the token task)